README.md

```diff
@@ -254,6 +254,7 @@
 
 - 💙🧩 [Tailwind UI](https://tailwindui.com) - Component library made with Tailwind CSS.
 - 💙📚 [Headless UI](https://github.com/tailwindlabs/headlessui) - Completely unstyled, fully accessible UI components.
+- 📚 [Vanilla Components](https://vanilla-components.com) - A lightweight, flexible & customizable UI library for Vue 3, styled with Tailwind CSS.
```

Comment:
Should be below "Headless UI float". Also please adapt the description to match the guidelines.

Reply:
Thanks. I just submitted the requested changes.